DOCS-1615: Fixing issues found in testing (#760)
tigera/docs: calico/getting-started/kubernetes/kind.mdx

Kind cluster creation can be modified by [configuration file](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster). Here you are providing kind configuration in order to disable simple networking implementation (default CNI) and change the pod ip CIDR.

> 1. Disable the default CNI in order to use Calico CNI.

Reviewer: ```suggestion
1. Disable the default CNI in order to use Calico CNI by running the following command:
```

Author: Done

EOF
```

> 2. Start your Kind cluster with one control plane and two worker nodes using the following command.

Reviewer: ```suggestion
2. Start your Kind cluster with one control plane and two worker nodes by running the following command:
```

Author: Done

```bash
kind create cluster --config values.yaml --name dev
```

> 3. Confirm that you now have three nodes in your cluster with the following command.

Reviewer: ```suggestion
3. Confirm that you have three nodes in your cluster by running the following command:
```

Author: Done
